ProjectCoimbra.UWP/Project.Coimbra/Pages/GamePage.xaml.cs

```diff
@@ -214,5 +214,37 @@ private void Timer_Tick(object sender, object e)
                 }
             }
         }
+
+        /// Only pause the playback in SOLO mode
+        private async void Pause_Click(object sender, RoutedEventArgs e)
+        {
+            if (UserData.GameMode == UserData.Mode.Solo)
+            {
+                midiEngine.Pause();
+            }
+            ContentDialog dialog = new ContentDialog
+            {
+                Title = "Game paused",
+                Content = "Do you want to continue the game?",
+                PrimaryButtonText = "Continue",
+                CloseButtonText = "Exit to Game Mode",
+                DefaultButton = ContentDialogButton.Primary
+            };
+
+            var result = await dialog.ShowAsync();
```

Comment:
After `ShowAsync()`, please add `.ConfigureAwait(false)` to more efficiently use async resources and help the program run more quickly. Test it afterwards, and if it doesn't work, use `.ConfigureAwait(true)` to make the intention clear and to ensure there are no problems later upgrading to .NET Core.

Reply:
Hi Muiris. 
There is no method `.ConfigureAwait()` for ContentDialog class.
I believe that  this `ShowAsync()` already handle this

Reply:
I think we're thinking of different things :smile:

I meant to do `await dialog.ShowAsync().ConfigureAwait(false)`. It should always be possible to add `.ConfigureAwait(false)` after an async method call.